server/tests/batch_comparison/snapshots/snap_test_batch_comparison.py

```diff
@@ -8,15 +8,15 @@
 snapshots = Snapshot()
 
 snapshots["test_batch_comparison_round_1 1"] = {
-    "numSamples": 14,
+    "numSamples": 10,
```

Comment:
why did these change?

Reply:
due to the changes i had to make in the test data in test_batch_comparison.py below.

the test in question was uploading the manifests and cvrs via standard fixtures and then updating the ballot manifest to something different than the test fixtures. since the new ballot manifest didn't match the cvrs, this code change caused the test to break.

the simplest fix was to remove the part of that test that changed the manifest. i confirmed that the modified manifest wasn't necessary. that lowered the sample size for that test, which changed these snapshots.